examples/minimal/hellodb.py

```
@@ -0,0 +1,62 @@
+logger = logging.getLogger(__name__)
+
+
+class DatabaseHelloTask:
+    """
+    A task definition for communicating with a database.
+    """
+
+    def __init__(self):
+        database_url = os.getenv("DATABASE_URL")
+        if database_url is None:
+            raise ValueError("Database URL environment variable is not set: DATABASE_URL")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)